static-instrumentation/agent-instrumenter/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id("otel.java-conventions")
+}
+
+ext.libraryName = 'OpenTelemetry Java Contrib Static Instrumentation Agent'
+description = 'OpenTelemetry Java Static Instrumentation Agent '
+
+jar {
+    manifest {
+        attributes('Main-Class': 'io.opentelemetry.javaagent.StaticInstrumenter')
```

Comment:
Let's either remove this or add an empty `StaticInstrumenter` class. Currently the java package here is not what we are targeting anyways

Reply:
🆗